devolutions-agent/src/updater/product_actions.rs

```diff
@@ -0,0 +1,88 @@
+use win_api_wrappers::service::{ServiceManager, ServiceStartupMode};
+
+use crate::updater::{Product, UpdaterError};
+
+const SERVICE_NAME: &str = "DevolutionsGateway";
```

Comment:
thought: I am under the impression that this const is already defined elsewhere. Could we reuse it?

Reply:
Hm, I didn't find it anywhere in the rust code, only in C# MSI installer code (service creation/removal logic) 🤔

Reply:
I was thinking of this:

https://github.com/Devolutions/devolutions-gateway/blob/2e3aa7dd459b538eaaeb6bd3f1260c71e632f441/devolutions-agent/src/service.rs#L19-L21

But it appears to be slightly different… Is it normal? :thinking:

Reply:
Ah.... I believe this const is used to create/start service via ceviche controller, strange that it have a different value.

The real service name which is created by MSI is `DevolutionsGateway`

Reply:
Maybe something to refactor here!